src/runtime/config/configuration-qemu-nvidia-gpu-snp.toml.in

```diff
@@ -52,7 +52,7 @@ sev_snp_guest = true
 # List of valid annotation names for the hypervisor
 # Each member of the list is a regular expression, which is the base name
 # of the annotation, e.g. "path" for io.katacontainers.config.hypervisor.path"
-enable_annotations = @DEFENABLEANNOTATIONS@
+enable_annotations = @DEFENABLEANNOTATIONSTEE@
```

Comment:
Shouldn't this be also reflected on the other config files?

Reply:
That sounds reasonable for the the gpu ones (gpu-snp, gpu-tdx and gpu).

The other ones seem to be taking the same direction (tdx and coco-dev), so maybe we can just update `@DEFENABLEANNOTATIONS@` with the new entries and get rid of `@DEFENABLEANNOTATIONSTEE@`.

Reply:
Changing the default_* will create a discrepancy with the podoverhead. Those two things need to go hand in hand.

Reply:
@zvonkok I think the proposal is not to change the `default_` but the `enable_`, so that sounds OK.